frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarSessionSummary.tsx

```diff
@@ -405,12 +407,121 @@ function SessionSummaryLoadingState({ operation, counter, name, outOf }: Session
     )
 }
 
-function SessionSummary(): JSX.Element {
-    const { logicProps } = useValues(sessionRecordingPlayerLogic)
+function SessionSummaryRoot({ children }: { children: React.ReactNode }): JSX.Element {
+    return <div className="flex flex-col">{children}</div>
+}
+
+function SessionSummaryTitle(): JSX.Element {
+    return (
+        <h3 className="text-lg font-semibold mt-2 flex items-center gap-2">
+            <IconAIText />
+            AI Replay Research
+            <LemonTag type="completion" size="medium">
+                ALPHA
+            </LemonTag>
+        </h3>
+    )
+}
+
+function SessionSummarySubtitle({ sessionId }: { sessionId: string }): JSX.Element {
+    return (
+        <div className="flex align-center text-md gap-1">
+            <p className="text-md ">Session ID: </p>
+            <Tooltip title="View recording">
+                <Link to={urls.replaySingle(sessionId)} target="_new">
+                    {sessionId}
+                </Link>
+            </Tooltip>
+        </div>
+    )
+}
+
+function SessionSummaryOutcomeBanner({ sessionSummary }: { sessionSummary: SessionSummaryContent }): JSX.Element {
```

Comment:
Nitpick: relying just on `success` is itchy, as 100% properly generated summary should include `description` also (as it checked before). Passing without `description` could lead to processing incomplete summaries. Should be impossible now, because of no-streaming, so rather paranoid take, but still :)